typings/test_index.js

```diff
@@ -11,15 +10,15 @@ import {
   within,
   getQueriesForElement,
   getDefaultNormalizer,
-} from '../..';
+} from '@testing-library/react-native';
```

Comment:
@thymikee you suggested to export flow types the same way we did with TS, but I can't get the import here to work (hence the red build), are you familiar with how to do that?